### PR TITLE
Add `add_customer_negative_crierion` example.

### PR DESCRIPTION
**Pull request by BenRKarl**

### CODE REVIEW

File: examples/targeting/add_customer_negative_criteria.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    # 'http://www.example.com'.
+    placement_criterion.placement.url = "http://www.example.com"
+
+    customer_negative_criterion_service = client.get_service(
+        "CustomerNegativeCriterionService", version="v5"
+    )
+
+    try:
+        # Issues a mutate request to add the negative customer criteria.
+        response = customer_negative_criterion_service.mutate_customer_negative_criteria(
```
Comment (BenRKarl) [inline examples/targeting/add_customer_negative_criteria.py:59]: This line exceeds the char limit, and newlining it doesn't help either. I didn't want to rename the vars so leaving as-is seemed like the most readable option.

File: examples/targeting/add_customer_negative_criteria.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    # of 'TRAGEDY'.
+    tragedy_criterion.content_label.type = client.get_type(
+        "ContentLabelTypeEnum", version="v5"
+    ).TRAGEDY
+
+    placement_criterion_op = client.get_type(
+        "CustomerNegativeCriterionOperation", version="v5"
+    )
+    placement_criterion = placement_criterion_op.create
+    # Creates a negative customer criterion excluding the placement with url
```
Comment (AndrewMBurke) [inline examples/targeting/add_customer_negative_criteria.py:49]: Nit: `URL`

**Review by AndrewMBurke — APPROVED** [review]
LGTM with one minor suggestion.

File: examples/targeting/add_customer_negative_criteria.py (hunk 1 above)
Comment (wihl) [reply examples/targeting/add_customer_negative_criteria.py:59]: how about something like :
```
response = (customer_negative_criterion_service
                       .mutate_customer_negative_criteria(customer_id, 
                           [tragedy_criterion_op, placement_criterion_op]
                    ))
```

File: examples/targeting/add_customer_negative_criteria.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    )
+
+    try:
+        # Issues a mutate request to add the negative customer criteria.
+        response = customer_negative_criterion_service.mutate_customer_negative_criteria(
+            customer_id, [tragedy_criterion_op, placement_criterion_op]
+        )
+        print(f"Added {len(response.results)} negative customer criteria:")
+        for negative_criterion in response.results:
+            print(negative_criterion.resource_name)
```
Comment (wihl) [inline examples/targeting/add_customer_negative_criteria.py:64]: Shouldn't this be `print(f"Resource name: \"{negative_criterion.resource_name}\"")` ?

File: examples/targeting/add_customer_negative_criteria.py (hunk 1 above)
Comment (BenRKarl) [reply examples/targeting/add_customer_negative_criteria.py:59]: Same as in the other PR, the auto-formatter will undo it and use a style that's longer than 88 chars.

File: examples/targeting/add_customer_negative_criteria.py (hunk 2 above)
Comment (BenRKarl) [reply examples/targeting/add_customer_negative_criteria.py:49]: Done.

File: examples/targeting/add_customer_negative_criteria.py (hunk 3 above)
Comment (BenRKarl) [reply examples/targeting/add_customer_negative_criteria.py:64]: Yeah that works, updated, though I used single-quotes per our offline discussion.

File: examples/targeting/add_customer_negative_criteria.py [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    )
+
+    try:
+        # Issues a mutate request to add the negative customer criteria.
+        response = customer_negative_criterion_service.mutate_customer_negative_criteria(
+            customer_id, [tragedy_criterion_op, placement_criterion_op]
+        )
+        print(f"Added {len(response.results)} negative customer criteria:")
+        for negative_criterion in response.results:
+            print(f"Resource name: {negative_criterion.resource_name}'")
```
Comment (wihl) [inline examples/targeting/add_customer_negative_criteria.py:64]: missing leading '

**Review by wihl — APPROVED** [review]
LGTM with one minor change needed

**Pull request merged**